internal/scheduler/jobs/anchor_platform_auth_enforcement_job.go

```diff
@@ -0,0 +1,76 @@
+package jobs
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/anchorplatform"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/crashtracker"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/monitor"
+)
+
+const (
+	apAuthMonitoringJobName            = "anchor_platform_auth_monitoring_job"
+	apAuthMonitoringJobIntervalSeconds = 300
+)
+
+type AnchorPlatformAuthMonitoringJob struct {
+	apService          anchorplatform.AnchorPlatformAPIServiceInterface
+	monitorService     monitor.MonitorServiceInterface
+	crashTrackerClient crashtracker.CrashTrackerClient
+}
+
+// NewPaymentService creates a new PaymentService
+func NewAnchorPlatformAuthMonitoringJob(apService anchorplatform.AnchorPlatformAPIServiceInterface, monitorService monitor.MonitorServiceInterface, crashTrackerClient crashtracker.CrashTrackerClient) (*AnchorPlatformAuthMonitoringJob, error) {
+	if apService == nil {
+		return nil, fmt.Errorf("apService cannot be nil")
+	}
+	if monitorService == nil {
+		return nil, fmt.Errorf("monitorService cannot be nil")
+	}
+	if crashTrackerClient == nil {
+		return nil, fmt.Errorf("crashTrackerClient cannot be nil")
+	}
+
+	return &AnchorPlatformAuthMonitoringJob{
+		apService:          apService,
+		monitorService:     monitorService,
+		crashTrackerClient: crashTrackerClient,
+	}, nil
+}
+
+func (job AnchorPlatformAuthMonitoringJob) GetInterval() time.Duration {
+	return apAuthMonitoringJobIntervalSeconds * time.Second
+}
+
+func (job AnchorPlatformAuthMonitoringJob) GetName() string {
+	return apAuthMonitoringJobName
+}
+
+func (job AnchorPlatformAuthMonitoringJob) Execute(ctx context.Context) error {
+	log.Ctx(ctx).Infof("executing AnchorPlatformAuthMonitoringJob ...")
```

Comment:
Can we switch this to Debug ?

Reply:
Addressed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/10/commits/0a380e30939dd57210931c2405f26a8d09ff5986.